<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.nature.com/articles/d41586-019-03536-x"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="article:published_time" content="2019-11-18T00:00:00+00:00"/>
    <meta property="og:title" content="Got mutation? ‘Base editors’ fix genomes one nucleotide at a time"/>
    <meta property="og:description" content="A new class of CRISPR-based tools efficiently corrects point mutations in cell lines, animal models and perhaps the clinic."/>
  </head>
  <body>
    <article>
      <h1>Got mutation? ‘Base editors’ fix genomes one nucleotide at a time</h1>
      <h2>A new class of CRISPR-based tools efficiently corrects point mutations in cell lines, animal models and perhaps the clinic.</h2>
      <address><time datetime="2019-11-18T00:00:00+00:00">18 Nov 2019</time> by <a rel="author">Sandeep Ravindran</a></address>
      <figure>
        <img src="https://media.nature.com/w800/magazine-assets/d41586-019-03536-x/d41586-019-03536-x_17373716.jpg"/>
        <figcaption>Credit: Getty</figcaption>
      </figure>
      <p>When Xingxu Huang began thinking about correcting disease-causing mutations in the human genome, his attention turned naturally to CRISPR–Cas9. But it quickly became clear that the popular gene-editing tool wasn’t ideal for the majority of human disease mutations, which result from errors in single DNA nucleotides known as point mutations. More than 31,000 such mutations in the human genome are known to be associated with human genetic diseases. But CRISPR is not particularly efficient at correcting them.</p>
      <p>Then Huang learnt about base editors, a new class of genome-modifying proteins that excel at single-site mutations.</p>
      <p>Base editors chemically change one DNA base to another without completely breaking the DNA backbone. The first cytosine base editor (CBE), which chemically converts a cytosine–guanine (C–G) base pair into a thymine–adenine (T–A) base pair at a targeted genomic location, was developed in 2016 by chemical biologists David Liu and Alexis Komor at Harvard University in Cambridge, Massachusetts [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR1">1</a></sup>]. Another researcher in Liu’s laboratory, Nicole Gaudelli, developed the first adenine base editor (ABE) a year later [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR2">2</a></sup>]; it chemically transforms A–T to G–C base pairs.</p>
      <p>“Base editing gives very, very good efficiency, about 40–50% efficiency for cell lines,” says Huang, a geneticist at ShanghaiTech University in China. “That’s very high efficiency compared with traditional genome editing,” which is only one-tenth as efficient, he says.</p>
      <p>But base editors are not just more efficient than CRISPR–Cas9; they also cause fewer errors. CRISPR–Cas9 acts as molecular scissors that cut both strands of DNA. As the cell repairs the break, random bases can be inserted or deleted (indels), altering the gene sequence. Large chromosomal segments might even be deleted or rearranged. By altering just a specific nucleotide without making double-stranded breaks, base editors cause fewer unwanted mistakes.</p>
      <p>Researchers have applied these tools across the evolutionary tree, from bacteria and yeast to rice, wheat, zebrafish, mice, rabbits and monkeys. They have used them to knock out genes, and to create and correct animal models. They have applied them in very early human embryos in the laboratory. And they might one day use base editors to treat human genetic diseases.</p>
      <p>First, however, researchers have to overcome some key hurdles. Like CRISPR–Cas9, base editors sometimes edit sites other than their target. They are limited in which genomic regions they can edit and what base conversions they can perform. And if they are ever to be used in the clinic, researchers will have to get better at delivering them into tissues.</p>
      <p>But improved editors are being developed at a rapid rate. “It’s really a testament to how fast researchers have made progress in the field that we now have dozens of base editors that offer expanded targeting scope, improved DNA specificity and reduced off-target activity,” says Liu. His base-editor constructs have been sent out to more than 1,000 laboratories around the world, he says, and new papers that use these and related tools appear almost weekly.</p>
      <h3>Building an editor</h3>
      <p>To create the first base editor, Komor took advantage of a naturally occurring enzyme called APOBEC1. This enzyme, which is part of the cytidine deaminase family, chemically converts C to uracil (U), an analogue of T that occurs in RNA. Komor fused rat APOBEC1 to a catalytically impaired Cas9 nuclease that is unable to create DNA double-strand breaks. When a guide RNA directs the APOBEC1–Cas9 fusion protein to a target site, the deaminase converts C to U. The cell’s DNA-repair system then fixes the resulting U–G mismatch by converting it into a U–A base pair, and ultimately to a T–A pair.</p>
      <p>Additional refinements improved the protein’s efficiency: these included swapping Cas9 for a Cas9 ‘nickase’ that cuts the G-containing strand, thus nudging the cell to replace the G rather than the U when repairing the U–G mismatch. “That extra modification boosted our efficiencies up to levels that we were happy with,” says Komor, who is now at the University of California, San Diego. Dubbed BE3, the resulting protein edits cellular DNA with almost a tenfold higher efficiency than CRISPR–Cas9 and with less than 1% indel formation.</p>
      <p>The first ABE was tougher to crack. No known naturally occurring enzymes could chemically convert A to G in DNA. “It was a pretty big ask to create an enzyme that didn’t exist and have it work very well,” Gaudelli says. Luckily for her, Liu’s lab had expertise in using microbes to achieve the rapid directed evolution of proteins. Over seven rounds of evolution and protein engineering, Gaudelli gradually coaxed a bacterial enzyme called TadA, which converts A to G in some RNAs, to accept a DNA substrate and work better in mammalian cells, producing an editor called ABE7.10.</p>
      <p>Although they can effect only a subset of possible nucleotide changes, such enzymes can already address the majority of disease-causing point mutations in humans, at least in theory. “The adenine base editor, in particular, corrects the most common kind of point mutation in humans,” says Liu, referring to G–C to A–T mutations, which account for about half of all known pathogenic single-nucleotide changes. For the moment, however, the technology is for laboratory use only.</p>
      <h3>Correcting and creating mutations</h3>
      <p>In initial studies, Liu’s team showed that CBEs could correct point mutations associated with Alzheimer’s disease and cancer [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR1">1</a></sup>] in mouse and human cell lines with an on-target editing efficiency of 35–75% and a 5% indel rate, compared with CRISPR–Cas9’s 0.1–0.3% efficiency and 26–40% rate of indel formation. Using ABEs, Liu’s team corrected point mutations responsible for a life-threatening blood-cell disorder called hereditary haemochromatosis, as well as sickle-cell anaemia [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR2">2</a></sup>].</p>
      <p>Researchers have used base editors to create and correct animal models of human diseases, including Duchenne muscular dystrophy [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR3">3</a></sup>] [<sup>–</sup>] [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR5">5</a></sup>], progeria [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR3">3</a></sup>] and age-related macular degeneration (H. Yang, unpublished observations). “With base editors, it’s easy to create an animal model and explore pathogenic mutations all over the genome,” says Huang, who has generated mouse models of diseases such as androgen insensitivity syndrome and syndactyly, a condition in which multiple fingers or toes are fused together [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR6">6</a></sup>]. Huang was even able to combine CBEs and ABEs in the same mouse embryos, resulting in simultaneous A–G and C–T edits, a trick he achieved using editors with different sequence preferences. “We can handle several mutations simultaneously and with very high efficiencies,” he says.</p>
      <p>Base editors can also be used to produce gene knockouts. The CRISPR–Cas9 system is particularly adept at creating knockouts, thanks to the natural mechanism most commonly used to repair double-strand DNA breaks. That process can add or delete bases at the cut site, causing the gene sequence to be misread and causing protein synthesis to stop prematurely. But CBEs can convert certain codons — the three-base genetic words that define the sequence of amino acids in a protein — to a stop signal directly, an idea that researchers are exploiting to systematically test the effects of knocking out different genes across the genome [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR7">7</a></sup>] [<sup>,</sup>] [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR8">8</a></sup>]. As base editors progress towards clinical trials, researchers have begun testing them in non-human primates. In unpublished work, Hui Yang, a developmental biologist at the Chinese Academy of Sciences in Shanghai, has applied base editors in mouse and monkey models of eye diseases, such as age-related macular degeneration, as well as Duchenne muscular dystrophy and Parkinson’s disease. “Base editors just cause single-strand breaks, not double-strand breaks, so I really think it’s more safe than CRISPR,” says Yang.</p>
      <p>Base editors could also be used to create high-yield or disease-resistant plant varieties, says Caixia Gao, a plant biologist at the Chinese Academy of Sciences in Beijing. “A single nucleotide change can make some rice plants better use nitrogen in the field, for example,” she says.</p>
      <h3>Building a better editor</h3>
      <p>Although theoretically similar to a genetic search-and-replace tool, base editors are in practice less precise.</p>
      <p>The fact that base editing uses Cas9 for sequence targeting means that it can produce off-target changes, just as CRISPR–Cas9 does. But base-editor specificity is complicated further by the deaminases that actually alter the DNA. These enzymes can modify RNA and single-stranded DNA at sites other than the intended target [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR9">9</a></sup>] [<sup>–</sup>] [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR11">11</a></sup>]. “We don’t know if these effects will be clinically relevant or not, but it’s wise to try to minimize any unwanted editing,” says Liu.</p>
      <p>ABEs apparently show no such off-target effects. This is probably because the ABE deaminase binds more weakly to its target than does the CBE deaminase, and so needs Cas9’s help for efficient editing, says Liu. Researchers have now developed higher-fidelity CBEs, such as HF-BE3, with weaker target binding, and found that they have correspondingly lower levels of off-target editing [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR12">12</a></sup>].</p>
      <p>Base editors can also sometimes edit ‘bystander’ Cs or As that lie within their ‘editing window’ — the nucleotide region within which the enzyme works efficiently. Researchers have created editors with narrower or broader windows to enhance or reduce such effects. For instance, YE1-BE3 and YEE-BE3 are modified versions of BE3 with narrower activity windows [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR13">13</a></sup>], whereas ABE7.9 [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR2">2</a></sup>] and the CBE BE-PLUS [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR14">14</a></sup>] have wider ones.</p>
      <p>“If we think about genetic disease correction, we need to have very specific editing, where we need to have this activity window be very narrow, down to one nucleotide,” says Gao. But an expanded editing window could be useful for accessing multiple target sites, for instance to introduce several point mutations at once.</p>
      <p>Base editors are also relatively limited in terms of the genomic sites that they can target; they can only act near a protospacer adjacent motif (PAM), the short DNA sequence required for successful binding of Cas9 to a DNA target. Because of that restriction, “I believe only about 25% of the pathogenic mutations in the human genome can be precisely edited or corrected using current tools”, says Huang. Researchers have expanded base editors’ scope by using directed evolution to create Cas9 proteins that recognize a broader range of PAMs, and by fusing base editors to Cas9 variants with wider PAM compatibility.</p>
      <p>And then there is the issue of the limited range of base changes that editors can currently produce. To correct as many genetic diseases as possible, base editors will need to perform additional conversions, such as C to A, C to G, A to C and A to T. Jin-Soo Kim, a biochemist at the Institute for Basic Science in Daejeon, demonstrated this year that ABEs can achieve C-to-G conversion as well as C-to-T and A-to-G conversions in a human kidney cell line [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR15">15</a></sup>]. “These results give us a hint on how to make other types of base editors,” he says.</p>
      <p>Alternatively, researchers could use a new class of genome editors from Liu’s lab, called prime editors, which can change any DNA base into any other [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR16">16</a></sup>]. Prime editors use a special guide RNA template and Cas9 nickase to direct a reverse transcriptase enzyme to a target site. There, the enzyme makes a new DNA strand from the RNA template and inserts it at the target (see ‘Prime corrective’). But there are a lot of unknowns with these tools, “including whether we can successfully do prime editing in animals and whether it will be as generalizable for many different types of cells as base editing”, says Liu.</p>
      <figure>
        <img src="https://media.nature.com/w800/magazine-assets/d41586-019-03536-x/d41586-019-03536-x_17387400.jpg"/>
        <figcaption>Source: Andrew Anzalone and David Liu</figcaption>
      </figure>
      <p>With all these different options, researchers will need to consider their needs carefully to find the best fit for their project. For efficiently disrupting genes or inserting or replacing large DNA sequences, CRISPR–Cas9 is the best bet, says Liu. It has been well studied, has lots of variants with greater specificity or particular PAM affinities, and is already being tested in clinical trials. Prime editors offer the greatest flexibility for creating DNA insertions, deletions, point mutations or combinations thereof. And base editors are ideal for correcting point mutations, providing higher efficiency and causing fewer indels.</p>
      <p>“I think all three of these classes of genome-editing agents really have complementary strengths and weaknesses,” says Liu. He likens CRISPR–Cas9 to scissors, base editors to pencils, and prime editors to word processors. “I think they all have their own roles in research and in applications such as agriculture and human therapeutics, just as scissors, pencils and word processors all have their own useful and unique roles.”</p>
      <h3>As easy as CRISPR</h3>
      <p>And just like scissors, pencils and word processors, base editing has been rapidly adopted by the scientific community, a testament to its low barrier to entry. “If you are familiar with genome-editing technology, I think you are ready to do base editing,” says Kim.</p>
      <p>Researchers can order base editors from the non-profit plasmid repository Addgene. Liu recommends starting with some of the newest editors, such as his lab’s BE4Max and ABEMax, which target C and A, respectively. But many others could also fit the bill, he adds, depending on the circumstances. (See Table 1 in ref. 17 for a good starting point.)</p>
      <p>Consider PAM specificity and the editing window required to access the target, Liu says. Consider also how much to prioritize reduced bystander editing or off-target effects. Specialized computational tools such as beditor can help researchers to design guide RNAs for their particular target.</p>
      <p>Still, base editors don’t always work as expected. “Sometimes we have to test a couple of different editors before we find one that likes our target,” says Komor. If nothing works, researchers can cut and paste from different base editors to make a custom editor, a process that Komor says is relatively straightforward. “Don’t be afraid to make your own.”</p>
      <p>Whatever the editor, delivering them to cells requires standard genetic techniques, such as transfection, micro-injection and electroporation. “You can deliver them as protein–RNA complexes, as mRNA or as DNA,” says Liu. Therapeutic applications, however, will require a different approach.</p>
      <p>Conventional viral delivery vectors, such as adeno-associated virus (AAV), carry only limited genetic cargo, and base editors are typically too large to fit. “Our current work is aimed at decreasing the size of the Cas9 and base editor, which I think will broaden its application,” says Yang. Alternatively, researchers can split base editors across two vectors, as Kim did to target a mutation in the Duchenne muscular dystrophy gene in adult mice. “We were able to correct the mutation in skeletal muscle,” he says [<sup><a href="https://www.nature.com/articles/d41586-019-03536-x#ref-CR5">5</a></sup>].</p>
      <p>It is early days, but base editors have already become a promising addition to the genome-editing toolset. And they might have more tricks up their sleeves. Some editors, for instance, can act on RNA rather than DNA, opening up the possibility of knocking down or editing mRNA transcripts containing pathogenic mutations. Base editors might also be able to target mutations in mitochondria, which lack the DNA-repair pathways that conventional genome editing relies on, says Kim.</p>
      <p>For Gaudelli, such opportunities represent the realization of a lifelong dream. “My motivation for being in the sciences was to make a difference in the world,” she says. “I never thought it would be through base editing.”</p>
      <p>Nature <b>575</b>, 553-555 (2019)</p>
      <p>doi: 10.1038/d41586-019-03536-x</p>
      <h3>References</h3>
      <p>1. <anchor name="ref-CR1"/>Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. A. &amp; Liu, D. R. <i>Nature</i> <b>533</b>, 420–424 (2016).</p>
      <p>2. <anchor name="ref-CR2"/>Gaudelli, N. M. <i>et al.</i> <i>Nature</i> <b>551</b>, 464–471 (2017).</p>
      <p>3. <anchor name="ref-CR3"/>Liu, Z. <i>et al.</i> <i>Nature Commun.</i> <b>9</b>, 2338 (2018).</p>
      <p>4. <anchor name="ref-CR4"/>Liu, Z. <i>et al.</i> <i>Nature Commun.</i> <b>9</b>, 2717 (2018).</p>
      <p>5. <anchor name="ref-CR5"/>Kim, K. <i>et al.</i> <i>Nature Biotechnol.</i> <b>35</b>, 435–437 (2017).</p>
      <p>6. <anchor name="ref-CR6"/>Ryu, S.-M. <i>et al.</i> <i>Nature Biotechnol.</i> <b>36</b>, 536–539 (2018).</p>
      <p>7. <anchor name="ref-CR7"/>Kuscu, C. <i>et al.</i> <i>Nature Methods</i> <b>14</b>, 710–712 (2017).</p>
      <p>8. <anchor name="ref-CR8"/>Billon, P. <i>et al.</i> <i>Mol. Cell</i> <b>67</b>, 1068–1079 (2017).</p>
      <p>9. <anchor name="ref-CR9"/>Grünewald, J. <i>et al.</i> <i>Nature</i> <b>569</b>, 433–437 (2019).</p>
      <p>10. <anchor name="ref-CR10"/>Zuo, E. <i>et al.</i> <i>Science</i> <b>364</b>, 289–292 (2019).</p>
      <p>11. <anchor name="ref-CR11"/>Jin, S. <i>et al.</i> <i>Science</i> <b>364</b>, 292–295 (2019).</p>
      <p>12. <anchor name="ref-CR12"/>Rees, H. A. <i>et al.</i> <i>Nature Commun.</i> <b>8</b>, 15790 (2017).</p>
      <p>13. <anchor name="ref-CR13"/>Kim, Y. B. <i>et al.</i> <i>Nature Biotechnol.</i> <b>35</b>, 371–376 (2017).</p>
      <p>14. <anchor name="ref-CR14"/>Jiang, W. <i>et al.</i> <i>Cell Res.</i> <b>28</b>, 855–861 (2018).</p>
      <p>15. <anchor name="ref-CR15"/>Kim, H. S., Jeong, Y. K., Hur, J. K., Kim, J.-S. &amp; Bae, S. <i>Nature Biotechnol.</i> <b>37</b>, 1145–1148 (2019).</p>
      <p>16. <anchor name="ref-CR16"/>Anzalone, A. V. <i>et al.</i> <i>Nature</i> https://doi.org/10.1038/s41586-019-1711-4 (2019).</p>
      <p>17. <anchor name="ref-CR17"/>Rees, H. A. &amp; Liu, D. R. <i>Nature Rev. Genet.</i> <b>19</b>, 770–788 (2018).</p>
      <related>
        <a href="http://www.nature.com/articles/s41586-019-1765-3"/>
        <a href="http://www.nature.com/articles/s41586-019-1775-1"/>
        <a href="http://www.nature.com/articles/s41586-019-1763-5"/>
        <a href="http://www.nature.com/articles/d41586-019-03525-0"/>
        <a href="http://www.nature.com/articles/d41586-019-03522-3"/>
        <a href="http://www.nature.com/articles/d41586-019-03392-9"/>
        <a href="http://www.nature.com/articles/s41586-019-1768-0"/>
        <a href="http://www.nature.com/articles/s41586-019-1744-8"/>
      </related>
    </article>
  </body>
</html>