<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <meta property="og:url" content="https://www.sciencedaily.com/releases/2019/11/191118140339.htm"/>
    <meta property="og:site_name" content="ScienceDaily"/>
    <meta property="article:published_time" content="2019-11-18T00:00:00+00:00"/>
    <meta property="og:title" content="Researchers clear the path for 'designer' plants"/>
    <meta property="og:description" content="A team of researchers has found a way to identify gene regulatory elements that could help produce 'designer' plants and lead to improvements in food crops at a critical time."/>
  </head>
  <body>
    <article>
      <h1>Researchers clear the path for 'designer' plants</h1>
      <h2>New targets for gene editing could lead to more resilient crops</h2>
      <address>
        <time datetime="2019-11-18T00:00:00+00:00">18 Nov 2019</time>
      </address>
      <table bordered="">
        <tr>
          <td>Source:</td>
          <td>University of Georgia</td>
        </tr>
        <tr>
          <td>Summary:</td>
          <td>A team of researchers has found a way to identify gene regulatory elements that could help produce 'designer' plants and lead to improvements in food crops at a critical time.</td>
        </tr>
      </table>
      <p>
        <b>A team of researchers at the University of Georgia has found a way to identify gene regulatory elements that could help produce "designer" plants and lead to improvements in food crops at a critical time. They published their findings in two separate papers in <i>Nature Plants</i>.</b>
      </p>
      <p>With the world population projected to reach 9.1 billion by 2050, world food production will need to rise by 70% and food production in the developing world will need to double, according to estimates from the Food and Agricultural Organization of the United Nations. Improvements in crop plants could play a key role in that effort.</p>
      <p>The team, led by Bob Schmitz, demonstrated an ability to identify cis-regulatory elements, or CREs, in 13 plant species, including maize, rice, green beans and barley.</p>
      <p>Cis-regulatory elements are regions of noncoding DNA that regulate neighboring genes. If a gene and its CRE can be identified, they can be treated as a modular unit, sometimes called a biobrick. Targeting CREs for editing offers a more refined tool than editing genes, according to Schmitz, associate professor of genetics in the Franklin College of Arts and Sciences.</p>
      <p>"Gene editing can be like a hammer. If you target the gene, you pretty much break it," he said. "Targeting CREs, which are involved in controlling gene expression — how a particular characteristic appears — allows you to turn gene expression up or down, similar to a dial. It gives us a tool to create a whole range of variation in expression of a gene."</p>
      <p>Controlling a gene for leaf architecture, for example, might allow a plant breeder to choose the angle at which a leaf grows from a plant, which can play a significant role in the plant's light absorption and growth. Targeting the gene itself would provide two options: "on," where the leaf might grow at a 90-degree angle, and "off," where the leaf might grow straight down. But targeting the CRE instead of the gene would allow the grower to target a range of possibilities in between — a 10-degree angle, a 25-degree angle, a 45-degree angle, etc.</p>
      <p>Once biobricks have been created and screened for the desired output, they could be used to produce "designer" plants that possess desirable characteristics — for example, salt-tolerant plants that can grow in a landscape with high salinity. The ability to design plants to grow in less-than-ideal landscapes will become more and more important as food growers strive to produce more in an environment facing increasing challenges, like drought and flooding.</p>
      <p>Based on their success, the research team recently received a $3.5 million grant from the National Science Foundation to investigate the role of CREs in legumes, including peanuts and soybeans.</p>
      <p>Underlying the grant proposal and the papers are technological breakthroughs developed by Zefu Lu, Bill Ricci and Lexiang Ji.</p>
      <p>"Zefu took a high-throughput method for identifying specific elements that was developed for animal cells and found a way to apply it to plant cells. It took a long time to address the significant barrier of plant organellar genomes, but now we're able to do what the animal field has been doing for a few years," Schmitz said.</p>
      <p>"When people try to find trait/disease associations, they look for mutations in genes, but the work in animals has shown that these non-gene regions also possess mutations that affect the way in which a gene is expressed. The regions we're identifying with this method are revealing regulatory information for gene expression control, which traditionally has been challenging to detect compared to genes."</p>
      <p>One of Ricci's contributions was developing a technique that shows the link between CREs and the gene they control.</p>
      <p>"Typically CREs are located right next to the gene they control, but in plants with larger genomes — soybeans, maize — it's become clear that these controlling elements can appear very far away," Schmitz said. "In two-dimensional space something may appear far away, over many thousands of base pairs, but Bill's method shows that in three dimensions, it's actually positioned right next to the gene."</p>
      <p>This work — the first time it has been applied to plants — provided the foundation for the two papers published in <i>Nature Plants</i>, and Schmitz paid tribute to his team members' contributions.</p>
      <p>"This is a group effort," he said. "Zefu, Bill and Lexiang were major drivers of this research."</p>
      <p>"Widespread Long-range Cis-Regulatory Elements in the Maize Genome" provides genetic, epigenomic and functional molecular evidence supporting the widespread existence of long-distance loci that act as long-range CREs influencing if and how a gene in the maize genome is expressed.</p>
      <p>In "The prevalence, evolution and chromatin signatures of plant regulatory elements," the researchers identified thousands of CREs and revealed that long-distance CREs are prevalent in plants, especially in species with large and complex genomes. Additional results suggest that CREs function with distinct chromatin pathways to regulate gene expression.</p>
      <p>The team's work will be shared via publicly available epigenome browsers that were developed by Brigitte Hofmeister, a recent Ph.D. graduate from the Schmitz Lab.</p>
      <p>"Our studies are genome wide, and we do a lot of technique and technology development, but it's not useful if people can't access it," Schmitz said. "We provide epigenome browsers that allow people studying leaf architecture, for example, to access information on the specific genes or traits they're interested in."</p>
      <p>Industry is also interested in CREs, according to Schmitz. Their editing pipeline is well established for genes, and the next obvious target for editing is CREs once they are located.</p>
      <p>"It's not just academia using this for basic science," he said. "The applications of this approach to identify CREs will become commonplace in industry to improve crop performance."</p>
      <p>make a difference: sponsored opportunity</p>
      <details open="">
        <summary>
          <b>Story Source:</b>
        </summary>
        <p><a href="https://news.uga.edu/researchers-clear-path-for-designer-plants/">Materials</a> provided by <a href="http://www.uga.edu/"><b>University of Georgia</b></a>. <br/><i>Note: Content may be edited for style and length.</i></p>
      </details>
      <details open="">
        <summary><b>Journal References</b>:</summary>
        <ol>
          <li>William A. Ricci, Zefu Lu, Lexiang Ji, Alexandre P. Marand, Christina L. Ethridge, Nathalie G. Murphy, Jaclyn M. Noshay, Mary Galli, María Katherine Mejía-Guerra, Maria Colomé-Tatché, Frank Johannes, M. Jordan Rowley, Victor G. Corces, Jixian Zhai, Michael J. Scanlon, Edward S. Buckler, Andrea Gallavotti, Nathan M. Springer, Robert J. Schmitz, Xiaoyu Zhang. <b>Widespread long-range cis-regulatory elements in the maize genome</b>. <i>Nature Plants</i>, 2019; DOI: <a href="http://dx.doi.org/10.1038/s41477-019-0547-0">10.1038/s41477-019-0547-0</a></li>
          <li>Zefu Lu, Alexandre P. Marand, William A. Ricci, Christina L. Ethridge, Xiaoyu Zhang, Robert J. Schmitz. <b>The prevalence, evolution and chromatin signatures of plant regulatory elements</b>. <i>Nature Plants</i>, 2019; DOI: <a href="http://dx.doi.org/10.1038/s41477-019-0548-z">10.1038/s41477-019-0548-z</a></li>
        </ol>
      </details>
      <details>
        <summary><b>Cite This Page</b>:</summary>
        <table bordered="">
          <tr>
            <td>
              <a href="https://www.sciencedaily.com/releases/2019/11/191118140339.htm#citation_mla">MLA</a>
            </td>
            <td>University of Georgia. "Researchers clear the path for 'designer' plants: New targets for gene editing could lead to more resilient crops." ScienceDaily. ScienceDaily, 18 November 2019. &lt;www.sciencedaily.com/releases/2019/11/191118140339.htm&gt;.</td>
          </tr>
          <tr>
            <td>
              <a href="https://www.sciencedaily.com/releases/2019/11/191118140339.htm#citation_apa">APA</a>
            </td>
            <td>University of Georgia. (2019, November 18). Researchers clear the path for 'designer' plants: New targets for gene editing could lead to more resilient crops. <i>ScienceDaily</i>. Retrieved November 19, 2019 from www.sciencedaily.com/releases/2019/11/191118140339.htm</td>
          </tr>
          <tr>
            <td>
              <a href="https://www.sciencedaily.com/releases/2019/11/191118140339.htm#citation_chicago">Chicago</a>
            </td>
            <td>University of Georgia. "Researchers clear the path for 'designer' plants: New targets for gene editing could lead to more resilient crops." ScienceDaily. www.sciencedaily.com/releases/2019/11/191118140339.htm (accessed November 19, 2019).</td>
          </tr>
        </table>
      </details>
      <related>
        <h4>Related Stories</h4>
        <a href="https://www.sciencedaily.com/releases/2019/05/190502143513.htm"/>
        <a href="https://www.sciencedaily.com/releases/2019/03/190321152841.htm"/>
        <a href="https://www.sciencedaily.com/releases/2016/11/161117150333.htm"/>
        <a href="https://www.sciencedaily.com/releases/2015/02/150219160514.htm"/>
      </related>
      <related>
        <h4>Related Terms</h4>
        <a href="https://www.sciencedaily.com/terms/lead.htm"/>
        <a href="https://www.sciencedaily.com/terms/plant_breeding.htm"/>
        <a href="https://www.sciencedaily.com/terms/chloroplast.htm"/>
        <a href="https://www.sciencedaily.com/terms/chromosome.htm"/>
        <a href="https://www.sciencedaily.com/terms/botany.htm"/>
        <a href="https://www.sciencedaily.com/terms/agriculture.htm"/>
        <a href="https://www.sciencedaily.com/terms/gene.htm"/>
        <a href="https://www.sciencedaily.com/terms/agronomy.htm"/>
      </related>
    </article>
  </body>
</html>